CONTRIBUTING.md

```diff
@@ -1,129 +1,116 @@
+This project, and everyone participating in it, are governed by the [Strapi Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold it. Make sure to read the [full text](CODE_OF_CONDUCT.md) to understand which type of actions may or may not be tolerated.
 
 ## Contributor License Agreement (CLA)
 
-### Individual
+### Individual contribution
 
-In order to accept your pull request, we need you to submit a CLA. You only need to do this once. If you are submitting a pull request for the first time, you can complete your CLA [here](https://cla.strapi.io/strapi/strapi) or just submit a Pull Request and our CLA Bot will ask you to sign the CLA before merging your Pull Request.
+In order to accept your pull request, you need to submit a Contributor License Agreement (CLA). You only need to do this once. If you are submitting a pull request for the first time, you can complete your CLA [here](https://cla.strapi.io/strapi/strapi) or submit a pull request and our CLA bot will automatically ask you to sign before merging the pull request.
 
-### Company
+### Company contribution
 
 If you are making contributions to our repositories on behalf of your company, then we will need a Corporate Contributor License Agreement (CLA) signed. In order to do that, please contact us at [contributions@strapi.io](mailto:contributions@strapi.io).
 
 ## Documentation
 
-Pull requests relating to fixing documentation for the latest release should be directed towards the [documentation repo](https://github.com/strapi/documentation). Please see the documentation [contributing guide](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md) for more information on how to make the documentation pull request.
+Pull requests related to fixing documentation for the latest release should be directed towards the [documentation repository](https://github.com/strapi/documentation). Please follow the [documentation contributing guide](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md) for more information.
 
 ## Bugs
 
-We are using [GitHub Issues](https://github.com/strapi/strapi/issues) to manage our public bugs. We keep a close eye on this so before filing a new issue, try to make sure the problem does not already exist.
+Strapi is using [GitHub issues](https://github.com/strapi/strapi/issues) to manage bugs. We keep a close eye on them. Before filing a new issue, try to make sure your problem does not already exist.
 
 ---
 
 ## Before Submitting a Pull Request
 
-The core team will review your pull request and will either merge it, request changes to it, or close it.
+The Strapi core team will review your pull request and either merge it, request changes to it, or close it.
+
+## Contribution Prerequisites
+
+- You have [Node.js](https://nodejs.org/en/) at version >= v12 and <= v16 and [Yarn](https://yarnpkg.com/en/) at v1.2.0+ installed.
```

Comment:
To run the unit tests you need node 14 now we can increase the require version of contributing

Reply:
Wasn't updated :)